### PR TITLE
turned footer back on, added style to make button bigger

### PR DESCRIPTION
**Pull request by wdwatkins**

### CODE REVIEW

**Comment by wdwatkins** [comment]
image

**Comment by wdwatkins** [comment]
I put the style in the template html, I assume it could be moved to a css file if that's better.

File: viz.yaml [hunk 1]
```diff
@@ -70,7 +70,7 @@ fetch:
     start.date: "2016-10-5"
     depends: ["sites-data","precip-data"]
     reader: rds
-    refetch: true
+    refetch: FALSE
```
Comment (ldecicco-USGS) [inline viz.yaml]: should this be turned back on before merging?

Comment (wdwatkins) [reply viz.yaml]: If we're going to be rebuilding it for updates, yes.  Although it's unclear if it's actually working.

Comment (ldecicco-USGS) [reply viz.yaml]: Ah...something I might actually be able to contribute. I'm just opening up the Jenkin's build, and it's definitely failing. I'll try to figure out why....

Comment (wdwatkins) [reply viz.yaml]: I just meant that refetch may/may not be working as intended

Comment (ldecicco-USGS) [reply viz.yaml]: I think it is working though

**Comment by wdwatkins** [comment]
Footer controlled to same width as viz:
image

css could probably be set up more logically if someone wants to, I was messing around with things a good bit

**Pull request merged**